examples/active_fire_detection/README.md

```diff
@@ -0,0 +1,205 @@
+
+##### 2: Associating data with labels, space, and time
+Satellite images have two extremely crucial pieces of metadata that cannot be ignored during model development: 
+- Where was this image taken? 
+- At what time was this image taken? 
+
+This data is obtained partially through metadata attached to the image files and partially through separate resources issued by the satellite providers, meaning it's often difficult to identify and communicate a single source of truth as a team. 
+
+##### 3: Data quality
+Satellites can't see through clouds! They're also susceptible to having compromised portions due to interference, sun glint, and other unavoidable error sources generated by the curvature and rotation of the earth. 
+
+Taken together, these features should give insight into why model development on satellite data requires active, granular, version-controlled curation of datasets to produce optimal outcomes. 
+
+## Where we're going 
+In this tutorial, we'll transform a widely-cited dataset for wildfires detection from a 10GB Google Drive-hosted .zip file to a cloneable, well-organized [Oxen.ai](www.oxen.ai) data repository ready for model development.
+
+## The Dataset 
+We'll be walking through how to prepare [this excellent dataset](https://drive.google.com/drive/folders/1GIcAev09Ye4hXsSu0Cjo5a6BfL3DpSBm) (introduced in [this paper by Pereira et. al](https://arxiv.org/abs/2101.03409)) for space-based active fire detection.
+
+This dataset contains patches of imagery extracted from LANDSAT-8 satellite imagery across Earth, many of which contain portions of actively burning wildfires. 
+
+While the authors processed the full 1.6TB of imagery captured by LANDSAT-8 in August 2020 (targeting peak fire season), we're primarily interested in the excellent subset of data for which they manually created segmentation masks denoting active burn. 
+
+![An image showing extensive global landmass coverage of the LANDSAT-8 images included in the training set](images/landsat_patches.png)
+
+In addition to being a more manageable (12GB) dataset size for model fine-tuning and evaluation applications, these human-verified labels will give any downstream models the best possible signal with which to learn to detect fires in novel imagery.
+
+ ![A synthetic color rendering of a landsat image of a fire zone, showing extensive smoke but partially obscured by cloud cover](images/landsat_burn_plain.png)
+
+![The same image, with a red pixel mask denoting the burn area](images/landsat_burn_annotated.png)
+
+In addition to these ground-truth manual annotations, the researchers include 5 similar computationally generated segmentation masks taken from best-practice formulas across the remote sensing literature. These leverage various properties of the pixel values of various spectral channels in the imagery to determine likely fire zones. While less precise and validated than the manual annotations, these can also be a useful reference point for model validation.
```

Comment:
This sentence feels like a mouth full.

"In addition to these ground-truth manual annotations, the researchers include 5 auto generated segmentation masks that don't require human labeling. These leverage computational algorithms that determine ..."